### PR TITLE
Made cold brew coffee recipe

### PR DESCRIPTION
**Pull request by actuallyexeon**

### CODE REVIEW

**Comment by Morelcia** [comment]
Missing .md extension

File: src/cold-brew-coffee [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- ⏲️ Prep time: 2 - 5 min
+- 🍽️ Servings: 1 (maybe 2)
+- ☕ Size: 1 pint 
+
+## Ingredients
+
+- 1 cup (1/2 pint) of coffee (As strong as you want) | Needed
+- 1 cup (1/2 pint) of milk | Needed
+- 3 - 5 Ice Cubes | Needed
+- 1/2 or 1 tbsp | Optional
```
Comment (hallzy) [inline src/cold-brew-coffee:14]: 1/2 or 1 tbsp of what?

Comment (actuallyexeon) [reply src/cold-brew-coffee:14]: Oop, I meant sugar

Comment (hallzy) [reply src/cold-brew-coffee:14]:
```suggestion
- 1/2 or 1 tbsp of sugar | Optional
```

**Review by Knarkzel — APPROVED** [review]

**Review by DGKSK8LIFE — APPROVED** [review]

**Review by hallzy — CHANGES_REQUESTED** [review]
btw, looks like `sugar` hasn't been added to the ingredients list as mentioned above

**Comment by Holsterbau** [comment]
Is this really cold brewing coffee? To my understanding, [cold brewing](https://en.wikipedia.org/wiki/List_of_coffee_drinks#Cold_brew) is a specific term for a process of brewing coffee through cold extraction, as opposed through the much more common hot brewing process. A notable advantage of cold brewing is that the resulting brew has almost no acidity, unlike hot brewed coffee. This just looks like a [regular frappé](https://en.wikipedia.org/wiki/List_of_coffee_drinks#Iced).

**Comment by actuallyexeon** [comment]
Well most people call it a Cold Brew AFAIK

**Comment by Holsterbau** [comment]
I am just speaking from the perspective that if someone wishes to make `cold-brew-coffee.md` in the future about how to cold brew coffee, this recipe would take up the spot even though it isn't *really* cold brewed coffee. It should probably be renamed to `iced-coffee.md` or something of the kind.

**Comment by actuallyexeon** [comment]
Ok! I'll rename it then

**Review by DGKSK8LIFE — APPROVED** [review]
Ok good definition change if everyone's in agreeance with this.

**Pull request closed without merging**